### DISCUSSION TITLE
[Projects Beta Pain Point] Why can't I add new issues when grouping by milestone?

### DISCUSSION THREAD

**Discussion by sidneymbell** [community/community#12110] [Projects and Issues]
This would be extremely high value for my workflow / is currently my only pain point in using Projects Beta that I can't easily work around. Is there some technical reason this isn't possible?

**Comment by nichujie** [comment]
Also not able to add items when grouping by repositories. Would be nice to enable this!

**Comment by pierremrtn** [comment]
This is really missing
